Adyen/UI/Form/Items/SelectableFormItem/SelectableFormItemView.swift

```diff
@@ -0,0 +1,164 @@
+        imageView.adyen.round(using: item.style.imageStyle.cornerRounding)
+        imageView.clipsToBounds = true
+        imageView.widthAnchor.constraint(equalToConstant: iconSize.width).isActive = true
+        imageView.heightAnchor.constraint(equalToConstant: iconSize.height).isActive = true
+        imageView.accessibilityIdentifier = ViewIdentifierBuilder.build(scopeInstance: self, postfix: "checkmark")
+        imageView.isHidden = !item.isSelected
+
+        return imageView
+    }()
+
+    // MARK: - Content StackView
+
+    private lazy var contentStackView: UIStackView = {
+        let stackView = UIStackView(arrangedSubviews: [imageView, titleLabel, checkmarkImageView])
+        stackView.setCustomSpacing(16, after: imageView)
+        stackView.spacing = 8
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.preservesSuperviewLayoutMargins = true
+        stackView.isLayoutMarginsRelativeArrangement = true
+        stackView.setContentHuggingPriority(.required, for: .vertical)
+        stackView.axis = .horizontal
+        stackView.alignment = .center
+        stackView.distribution = .fill
+
+        return stackView
+    }()
+
+    // MARK: - Item Button
+
+    private lazy var itemButton: UIButton = {
+        let customButton = UIButton(type: .custom)
+        customButton.addTarget(self, action: #selector(didSelectItemButton), for: .touchUpInside)
+        customButton.preservesSuperviewLayoutMargins = true
+        customButton.translatesAutoresizingMaskIntoConstraints = false
+        customButton.accessibilityIdentifier = item.identifier.map {
+            ViewIdentifierBuilder.build(scopeInstance: $0, postfix: "button")
+        }
+
+        customButton.addSubview(contentStackView)
+        contentStackView.isUserInteractionEnabled = false
+        contentStackView.adyen.anchor(inside: customButton)
+
+        return customButton
+    }()
+
+    @objc internal func didSelectItemButton() {
+        item.selectionHandler?()
+    }
+
+    /// Initializes the selectable form item view.
+    public required init(item: SelectableFormItem) {
+        super.init(item: item)
+        backgroundColor = item.style.backgroundColor
+
+        addSubview(itemButton)
+
+        accessibilityIdentifier = item.identifier
+        accessibilityLabel = item.accessibilityLabel
+        isAccessibilityElement = true
+
+        updateIcon()
+        updateImageView(style: item.style)
+        configureConstraints()
+
+        preservesSuperviewLayoutMargins = true
+
+        itemButton.adyen.anchor(inside: self.layoutMarginsGuide)
+
+        observe(item.$isSelected) { [weak self] isSelected in
+            guard let self else { return }
+            self.checkmarkImageView.isHidden = !isSelected
+        }
+    }
+
+    override public func didMoveToWindow() {
+        super.didMoveToWindow()
+        updateIcon()
+    }
+
+    override public func layoutSubviews() {
```

Comment:
What is the reason we style imageview every time in `layoutSubviews`? It should be sufficient to do in once when the view is created.

Reply:
This is done in case there is orientation change.

Reply:
I see! But does the orientation change affect imageView corner radius?

Reply:
Sometimes the corner radius can be distorted on orientation change.